generators/app/index.js

```diff
@@ -363,9 +372,17 @@ module.exports = class extends BaseGenerator {
                         if (this.clientPackageManager === 'yarn') {
                             this.log(chalk.bold(`\nInstalling generator-jhipster@${this.jhipsterVersion} locally using yarn`));
                             this.yarnInstall();
+                            if (this.options['code-formatter']) {
+                                this.log(`${chalk.bold('\nFormatting code with Tslint')}\n please wait`);
+                                shelljs.exec('yarn lint --fix');
```

Comment:
This is an issue, we shoulnt be relying on this and prettier, what rules are prettier breaking? is there way to control them in prettier? if not we can consider changing the rules to align with prettier for the ones that are broken

Reply:
Prettier don't use the shorthand {value} to {value: value} that tslint expect
I have fixed most of the other rules (quotes, parentheses) now
But i have not found any rules about this kind of format in prettier : 
https://prettier.io/docs/en/options.html